core/shared/src/main/scala/zio/Cause.scala

```diff
@@ -552,25 +552,25 @@ object Cause extends Serializable {
    * Converts the specified `Cause[Option[E]]` to an `Option[Cause[E]]` by
    * recursively stripping out any failures with the error `None`.
    */
-  def sequenceCauseOption[E](c: Cause[Option[E]]): Option[Cause[E]] =
+  def flipCauseOption[E](c: Cause[Option[E]]): Option[Cause[E]] =
```

Comment:
Love this name change. However, it isn't binary backward compatible, so we need to move it to 2.0, right?

Reply:
Fixed now.